swerve100/src/main/java/frc/robot/subsystems/AHRSClass.java

```diff
@@ -0,0 +1,174 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import com.kauailabs.navx.frc.AHRS;
+
+import edu.wpi.first.util.sendable.Sendable;
+import edu.wpi.first.util.sendable.SendableBuilder;
+import edu.wpi.first.wpilibj.I2C;
+import edu.wpi.first.wpilibj.SerialPort;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+/** Add your docs here. */
+public class AHRSClass implements Sendable {
+    public boolean gyrosWorking = true;
```

Comment:
make gyrosWorking private and write an accessor that reads it (e.g. "getGyrosWorking").  if you make it public than anyone can read it but also anyone can *write* to it.